### ISSUE TITLE
bigtable: bttest does not handle RowsLimit

### ISSUE THREAD

**Issue by akhenakh** [GoogleCloudPlatform/gcloud-golang#319]
When applying a `bigtable.LimitRows()` to a bttest `ReadRows()` no limits are applied.

Here is a possible solution
```
diff --git a/google.golang.org/cloud/bigtable/bttest/inmem.go b/google.golang.org/cloud/bigtable/bttest/inmem.go
index 7559c35..8297cf4 100644
--- a/google.golang.org/cloud/bigtable/bttest/inmem.go
+++ b/google.golang.org/cloud/bigtable/bttest/inmem.go
@@ -262,7 +262,11 @@ func (s *server) ReadRows(req *btspb.ReadRowsRequest, stream btspb.Bigtable_Read
        }
        sort.Sort(byRowKey(rows))

-       for _, r := range rows {
+       for i, r := range rows {
+               if req.RowsLimit > 0 && i >= int(req.RowsLimit) {
+                       return nil
+               }
+
                if err := streamRow(stream, r, req.Filter); err != nil {
                        return err
                }
```

**Comment by garye** [comment]
Thanks, I'll put together a CL for this.

**Comment by garye** [comment]
Fixed via e9eae4f583465473e798f6fea7f46beb41ffff85
